dashboard/app/views/home/_homepage.html.haml

```diff
@@ -14,6 +14,51 @@
 
 .clear{style: "clear: both"}
 
+- if current_user
+  - if current_user.teacher?
+    #terms_reminder
+      - unless current_user.accepted_latest_terms?
+        = render template: 'api/accept_terms_reminder'
+
+#terms-modal.modal{'data-backdrop' => 'static', 'data-keyboard' => 'false', style: 'display: none;'}
```

Comment:
If this is all copied from terms_interstitial.html.haml, can we just render that partial? i.e.

`= render partial: 'layouts/terms_interstitial'`